src/Data/Fin/Properties.agda

```diff
@@ -17,6 +17,7 @@ open import Data.Fin.Base
 open import Data.Fin.Patterns
 open import Data.Nat.Base as ℕ using (ℕ; zero; suc; s≤s; z≤n; _∸_)
 import Data.Nat.Properties as ℕₚ
+open import Data.Nat.Properties.Base using (≤-pred)
```

Comment:
We are already importing `Data.Nat.Properties` here so would it not
make sense to use `≤-pred` via it? It is publically re-exported after
all!

Reply:
Indeed.  Will fix.